x/token/keeper/keeper_test.go

```diff
@@ -93,6 +93,7 @@ func (s *KeeperTestSuite) SetupTest() {
 	}
 	s.contractID = s.keeper.Issue(s.ctx, class, s.vendor, s.vendor, s.balance)
 
+
```

Comment:
Useless change (Disgrace the logs in that file)

Reply:
Oh, I missed when reverting. Thanks.